scico/linop/__init__.py

```diff
@@ -11,7 +11,17 @@
 
 # isort: off
 from scico._generic_operators import LinearOperator
-from ._linop import Diagonal, Identity, power_iteration, operator_norm, Sum, Slice, valid_adjoint
+from ._linop import (
+    Diagonal,
```

Comment:
No need to include `linear_operator_from_function`?

Reply:
Fixed.